### PR TITLE
docs: skip exposing `t` + general docs improvements

### PR DESCRIPTION
**Pull request by KATT**
Closes #

## 🎯 Changes

What changes are made in this PR? Is it a feature or a bug fix?

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: www/docs/server/procedures.md [hunk 1]
```diff
@@ -195,3 +219,92 @@ export const appRouter = t.router({
 
 export type AppRouter = typeof appRouter;
 ```
+
+## Reusable base procedures
```
Comment (KATT) [inline www/docs/server/procedures.md:223]: https://www-git-create-router-take-2-trpc.vercel.app/docs/v10/procedures#reusable-base-procedures

File: examples/next-prisma-starter-sqlite/src/server/routers/health.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { createRouter, baseProcedure } from '../trpc';
+import { router, baseProcedure } from '../trpc';
```
Comment (juliusmarminge) [inline examples/next-prisma-starter-sqlite/src/server/routers/health.ts:1]: In docs you've changed the name of the `baseProcedure` to just `procedure`, or in some cases `publicProcedure`. Should prob keep the examples consistent.

File: www/docs/server/merging-routers.md [hunk 3]
```diff
@@ -7,26 +7,26 @@ slug: /merging-routers
...
 // @filename: trpc.ts
 import { initTRPC } from '@trpc/server';
-export const t = initTRPC.create();
+const t = initTRPC.create();
 
 
-// ---cut---
+export const middleware = t.middleware;
+export const router = t.router;
+export const procedure = t.procedure;
```
Comment (juliusmarminge) [inline www/docs/server/merging-routers.md:20]: for coherency?
```suggestion
export const publicProcedure = t.procedure;
```

File: www/docs/nextjs/introduction.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ import { initTRPC, TRPCError } from '@trpc/server';
 const t = initTRPC.create();
 
 // Base router and procedure helpers
-export const createRouter = t.router;
+export const router = t.router;
 export const baseProcedure = t.procedure;
```
Comment (juliusmarminge) [inline www/docs/nextjs/introduction.md:111]: for coherency?
```suggestion
export const publicProcedure = t.procedure;
```

File: www/docs/server/procedures.md [hunk 5]
```diff
@@ -195,3 +219,92 @@ export const appRouter = t.router({
...
+
+// -------------------------------------------------
+// @filename: trpc.ts
+// -------------------------------------------------
+import { initTRPC, TRPCError } from '@trpc/server';
+import { Context } from './context';
+
+const t = initTRPC.context<Context>().create();
+
+
```
Comment (juliusmarminge) [inline www/docs/server/procedures.md:262]:
```suggestion
/**
 * Reusable middleware that checks if users are authenticated.
 **/
```

File: examples/next-prisma-starter-sqlite/src/server/routers/health.ts (hunk 2 above)
Comment (KATT) [reply examples/next-prisma-starter-sqlite/src/server/routers/health.ts:1]: Out for lunch now, feel free to poke around

I think best would be publicProcedure everywhere

**Pull request merged**